docs/_controls/ct-12.md

```diff
@@ -1,12 +1,39 @@
 ---
-doc-status: Pre-Draft
+doc-status: Draft
 sequence: 12
 type:
  - Preventative
 mitigates:
- - tr-9
+ -  TR‑2: Insufficient Access Control with Vector Store
```

Comment:
this section is an array of keys, referencing risks - see for example:

https://github.com/finos/ai-governance-framework/blob/main/docs/_mitigations/mi-3.md?plain=1#L8

you don't need to include the title., just provide the key.